site/.storybook/preview.jsx

```diff
@@ -49,9 +49,6 @@ export const parameters = {
       locales: "en-US",
     },
   },
-  actions: {
-    argTypesRegex: "^(on|handler)[A-Z].*",
-  },
```

Comment:
I was reading up on this, and I think I get how it works
Did this basically make it so that for any stories that had props that started with `on` or `handle` automatically received a Storybook action function as the input?